### ISSUE TITLE
Some questions of training parameters

### ISSUE THREAD

**Issue by nihaoxiaoli** [luzhang16/AR-CNN#20]
The pretrained model which authors have released is 'epoch 3', but the paper said that epoch has only two stages. At the same time, I checked the pth file 'aligned_rcnn_0_3_17783.pth' and the 'optimizer' showed that the learning rate 'lr' was 0.0004. But the paper is 0.005, and decay it by 0.1 for another 1 epoch. What is the best training epoch and learning rate? In addition, using the 'aligned_rcnn_0_3_17783.pth'  released by the author, the test MR is only 13, less than 9. I wonder if different versions of pytorch and CUDA will cause different results?

**Comment by nihaoxiaoli** [comment]
I found the mistake. When I modified the code to fast-rcnn-pytorch-1.0, I didn't change the parameter 'feat_stride' so that the result is wrong. After fixing some errors, the results can be reproduced by the pretrained model released by the author. At present, I am training this network. Although the accuracy is worse than that of the paper, the gap is not big. After that, I will release pytorch-1.9 version so that the code can be run on 3090 and other GPUs.
